src/main/java/com/quiz/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.quiz.service.impl;
+
+import com.quiz.model.User;
+import com.quiz.repository.UserRepository;
+import com.quiz.service.UserService;
+
+import java.util.List;
+
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    public UserServiceImpl(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @Override
+    public String createUser(User user) {
+        try {
+            userRepository.addUser(user);
+            return "User created successfully.";
+        } catch (Exception e) {
+            e.printStackTrace();
+            return "Error creating user.";
+        }    }
+
+    @Override
+    public List<User> getUsers() {
+        return userRepository.getAllUsers();
+    }
+
+    @Override
+    public User getUser(Long id) {
+        return userRepository.getUserById(id);
```

Comment:
what if there is no user matching the given id number? so i guess you should add if condition.